docs/changes/utils/11957.bugfix.rst

```diff
@@ -0,0 +1 @@
+Enable ``json.dump`` for FITS_rec with variable length (VLF) arrays.
```

Comment:
Can you remove this one? The change concerns `io.fits`.

Reply:
Should there be a test of json serialization, and if yes, should it be in `io.fits`, an if yes, would this change (only of the test) warrant change in `utils`?

Reply:
Ok, I made some set of tests.